gcs/src/main/scala/quasar/blobstore/gcs/SAAuthConfig.scala

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.blobstore.gcs
+
+import scala.Predef.String
+
+import argonaut._, Argonaut._
+
+import cats.implicits._
+
+import scala.{Array, Byte, StringContext, Either}
+
+import java.net.{URI, URISyntaxException}
+
+final case class GoogleAuthConfig(authCfg: ServiceAccountConfig) {
+  import GoogleAuthConfig.serviceAccountConfigCodecJson
+  val serviceAccountAuthBytes: Array[Byte] = authCfg.asJson.toString.getBytes("UTF-8")
```

Comment:
I think we can get rid of the class `GoogleAuthConfig` completely and add something like `val authBytes: Array[Byte] = asJson.toString.getBytes("UTF-8")` directly in `ServiceAccountConfig`

Reply:
what would we call `asJson` on once it's moved inside `ServiceAccountConfig`?

Reply:
something like:
```
  import GoogleAuthConfig.serviceAccountConfigCodecJson
  val jsonBytes = this.asJson.toString.getBytes("UTF-8")
```